server/src/main/scala/com/stratio/crossdata/server/CrossdataServer.scala

```diff
@@ -82,14 +82,17 @@ class CrossdataServer extends Daemon with ServerConfig {
             xdContext.getOrElse(throw new RuntimeException("Crossdata context cannot be started")),
             serverActorConfig)),
         actorName)
+      val broadcastActor=actorSystem.actorOf(ResourceManagerActor.props(Cluster(actorSystem),xdContext.get))
```

Comment:
This is silly but, wouldn't `broadcastActor` be better identified as `resourceMgrActor` or something of the kind?